plugins/utilities.json

```diff
@@ -119,6 +119,12 @@
         }
       ],
       "versions": {
+        "1.2.0": {
```

Comment:
Is there a reason this change is versioned as `1.2.0` (instead of `1.1.0`)?

Reply:
Check the file , it is written version 1.2 , so I think it is fine to update this mod.